### PR TITLE
Colorize battle buttons based on their states

### PR DESCRIPTION
**Pull request by completesomething**
This pull request ports over the battle list colorization from BAR so that there is a clearer separation between battle states with their sorting.

lobby

### CODE REVIEW

File: LuaMenu/widgets/chobby/components/battle/battle_list_window.lua [hunk 1]
```diff
@@ -637,6 +643,51 @@ function BattleListWindow:CompareItems(id1, id2)
...
+					battlebuttonstyle = {0.10, 0.10, 0.95, 0.65} --blue
+			else
+				if battle.isRunning then
+					battlebuttonstyle =  {0.70, 0.60, 0.1, 0.65} --yellow
+				else
+					battlebuttonstyle =  {0.10, 0.50, 0.10, 0.65} --green
+				end
+			end
+		end
+	end
```
Comment (sprunk) [inline LuaMenu/widgets/chobby/components/battle/battle_list_window.lua:673]: Why the pyramid?
```suggestion
	if battle.passworded then
		battlebuttonstyle =  {0.60, 0.10, 0.85, 0.65} --violet
	elseif battle.isMatchMaker then
		battlebuttonstyle =  {0.90, 0.10, 0.10, 0.65} --red
	elseif lobby:GetBattlePlayerCount(battleID) < 1 then
		battlebuttonstyle = {0.10, 0.10, 0.95, 0.65} --blue
	elseif battle.isRunning then
		battlebuttonstyle =  {0.70, 0.60, 0.1, 0.65} --yellow
	else
		battlebuttonstyle =  {0.10, 0.50, 0.10, 0.65} --green
	end
```

**Comment by completesomething** [comment]
It seems that the battle list buttons will flash with bright colors on initial open. I'm not sure how to resolve that.

**Comment by Licho1** [comment]
looks nice!

**Comment by completesomething** [comment]
I've adjusted the bk buttons to be more transparent so that the list is less colorful.

zklist

Anything preventing this from being merged?

**Comment by completesomething** [comment]
I guess this could be closed since the chobby skin I'm working on would supersede it.

**Pull request closed without merging**